### ISSUE TITLE
Missing deps @angular/platform-server & angularfire-lite

### ISSUE THREAD

**Issue by sebastien-yoobic** [CHBaker/router-animations#1]
Hi! Thanks for the repo, it helped me understood Angular router animations :) 
I noticed that 2 reps are missing in the package.json. 
- angularfire-lite
- @angular/platform-server
If you have errors with these reps just run: 
`npm i angularfire-lite && npm i @angular/platform-server`

**Comment by CHBaker** [comment]
That is correct, I didn't include them because that is only if you would like to host it with Firebase. You can run it locally if you like, but, I will add it to the deps just to make it easier to handle. Thanks @sebastien-yoobic, glad this repo helped
